File: tools/bench/answers/python_haiku/s3/01_rle.py
s = "aaabbcccca"
result = ""
i = 0
while i < len(s):
    char = s[i]
    count = 1
    while i + count < len(s) and s[i + count] == char:
        count += 1
    result += char + str(count)
    i += count
print(result)

File: tools/bench/answers/go_haiku/s1/01_rle.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	s := "aaabbcccca"
	var result string
	for i := 0; i < len(s); i++ {
		char := s[i]
		count := 1
		for i+1 < len(s) && s[i+1] == char {
			count++
			i++
		}
		result += string(char) + strconv.Itoa(count)
	}
	fmt.Println(result)
}
